sdk/cosmos/azure-cosmos-spark_3-1_2-12/databricks/databricks-jar-install.sh

```diff
@@ -2,10 +2,8 @@
 
 CLUSTER_NAME=$1
 JARPATH=$2
-JARFILE=$3
 [[ -z "$CLUSTER_NAME" ]] && exit 1
-[[ -z "$JARFILE" ]] && exit 1
-[[ -z "$JARFILE" ]] && exit 1
+[[ -z "$JARPATH" ]] && exit 1
```

Comment:
btw, could you please also rename the folder `azure-cosmos-spark_3-1_2-12/databricks`

I suspect end users may see this and may think it is a folder for sample code. how about renaming the folder to
`test-databricks` or something like that?